lib/markdownlint.js

```diff
@@ -922,11 +955,10 @@ function readConfig(file, parsers, callback) {
       return callback(new Error(message));
     }
     // Extend configuration
-    const configExtends = config.extends;
```

Comment:
Diff would be smaller without this change, too. I think before/after are pretty similar - let's limit this to what's needed?